src/vm/jitinterface.cpp

```diff
@@ -1945,6 +1945,62 @@ CEEInfo::getClassSize(
     return result;
 }
 
+unsigned
+CEEInfo::getHeapClassSize(
+    CORINFO_CLASS_HANDLE clsHnd)
+{
+    CONTRACTL{
+        SO_TOLERANT;
+        NOTHROW;
+        GC_NOTRIGGER;
+        MODE_PREEMPTIVE;
+    } CONTRACTL_END;
+
+    unsigned result = 0;
+
+    JIT_TO_EE_TRANSITION_LEAF();
+
+    TypeHandle VMClsHnd(clsHnd);
+    MethodTable* pMT = VMClsHnd.GetMethodTable();
+    _ASSERTE(pMT);
+    _ASSERTE(!pMT->IsValueType());
+    result = pMT->GetNumInstanceFieldBytes() + OBJECT_SIZE;
```

Comment:
I saw you comment `I changed this to GetNumInstanceFieldBytes() + OBJECT_SIZE to account for method table pointer.`. How about a comment to that effect here?

Reply:
Done.